doc/overview.rst

```diff
@@ -41,8 +41,9 @@ Requirements
 Installing :mod:`scikit-sparse` requires:
 
```

Comment:
That is a good idea. I tested Python 2.7, 3.4, 3.5 and 3.6 and NumPy 1.8 to 1.12. They are all supported. I did not test older versions because older versions are not supported by SciPy. The docs are updated.